index.html

```diff
@@ -6,6 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="description" content="" />
     <link rel="stylesheet" href="quiz.css" />
+    <script src="quiz.js"></script>
```

Comment:
Sam element script jest napisany wzorowo i podaje prawidłową ścieżkę do pliku. :thumbsup:

Ale uwaga! Jeśli umieścimy go w sekcji head możemy mieć kłopoty. 

Kod strony, wykonuje się od góry do dołu i jeśli natrafi od razu na plik js, będzie go próbował wykonać, zanim wyrenderują się wszystkie elementy strony. Na razie nic się nie stanie ale jeśli w pliku js ktoś napisze kawałek kodu odwołujący się do jakiegoś elementu strony, który jeszcze nie istnieje to będziemy mieć błędy...
W tej sytuacji można:
1. przenieść tag script na koniec strony, tuż przed zamknięciem elementu body
albo:
2. zostawić go tam gdzie jest ale dopisać atrybut defer. Dzięki temu przeglądarka wczytuje sobie plik w tle ale wykonuje go dopiero po załadowaniu całego dokumentu html
```<script src="quiz.js" defer>```

Reply:
Jasne! Z przyzwyczajenia z pracy na jQuery założyłam, że w pliku JS będziemy pisać kod w funkcji $( document ).ready(), co wówczas nie generowałoby żadnych problemów z wczytywaniem. Poprawka wysłana.